### PR TITLE
chore(examples): set remix packages' version to `'*'`

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
This way we don't have to manually update the version anymore + it's in line with our templates to be used with `create-remix`

### CODE REVIEW

File: examples/basic/package.json [hunk 1]
```diff
@@ -7,15 +7,15 @@
     "start": "remix-serve build"
   },
   "dependencies": {
-    "@remix-run/node": "1.6.4",
```
Comment (MichaelDeBoey) [inline examples/basic/package.json:10]: @mcansh This should have been at `1.7.0` btw, so it seems like the version script didn't run for some reason 🤔

**Review by machour — APPROVED** [review]
Heh, was thinking of proposing the same thing this morning while running an example locally.

**Comment by MichaelDeBoey** [comment]
@machour Great minds think alike 😉

**Comment by mcansh** [comment]
does this break "open in code sandbox"?

**Comment by MichaelDeBoey** [comment]
@mcansh No it doesn't

- https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/basic
- https://codesandbox.io/s/github/MichaelDeBoey/remix/tree/set-remix-packages-version/examples/basic

File: examples/basic/package.json (hunk 1 above)
Comment (mcansh) [reply examples/basic/package.json:10]: yeah changesets does the bumping now, but before we would always use `scripts/version.js` which would bump the examples

**Review by mcansh — APPROVED** [review]

**Pull request merged**